libs/frontend/feature-render/src/lib/compute-attrs-inputs-mixins.ts

```diff
@@ -29,11 +29,17 @@ export const computeAttrsInputsMixins = (
   const { renderMode, resolveTheme = true, editor } = options
 
   const c: IComponent = component
+  let r: IComponent | undefined = undefined
+  if (component.reusableSourceId) {
+    r = resolveComponent(context, c.reusableSourceId)
+  }
   const inputs: Record<string, unknown> = {}
   const attrs: Record<string, unknown> = {}
   const mixins: string[] = []
+  const content = c.content ?? r?.content
 
   if (editor) {
+    // TODO -- is this still used? pseudo styles still seem to work without it
```

Comment:
I noticed this while working on styles, it seems like we don't use the pseudo classes appended here anymore, but I'm not sure enough to remove the code yet.

Reply:
Yes, I think we still need this for non-default pseudo styles to work correctly due to how pseudo styles are added to `<style>` in the build mode (check [this comment](https://github.com/samatechtw/pubstudio-builder/pull/258#discussion_r1467651001) for more information).

https://github.com/samatechtw/pubstudio-builder/assets/59676941/b257891b-ae68-491b-8171-78c9f0637f86